docs-chef-io/content/server/upgrades.md

```diff
@@ -44,6 +44,38 @@ Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+### Upgrading to 14.8.x
+
+Chef Infra Server 14.8 upgraded PostgreSQL from 9.6 to 13.3. The 14.8 upgrade process requires downtime for vacuuming, upgrading and reindexing the database. There are extra steps that need to be considered when upgrading from versions less than 14.8 to versions greater than 14.8. Using current recommendations, the entire upgrade operation could take 1 minute per 1000 nodes or longer, depending on your server hardware and the complexity of your Chef data.
+
+#### Pre-upgrade Recommendations
+
+1. Running vacuum full is recommended if auto vacuuming is not set up, as this will reduce the size of the database by deleting unnecessary data. The vacuum full operation could take around 1 to 2 minutes per GB of data, depending on the nature of the data. Additionally, the vacuum full operation will need an amount of free disk space at least as large as the size of your database.
+
+    ```bash
+    /usr/lib/postgresql/9.6/bin/vaccumedb --all --full
+    ```
+
+2. Performing `chef-server-ctl backup` or `knife-ec-backup` is recommended before starting the upgrade. The backup takes around 4 to 5 minutes per GB of data.
+
+#### Upgrade Steps
+
+1. Follow the same instructions for upgrading to 14.
+
+#### Post-upgrade steps
+
+1. Reindex the Chef Infra server
+
+    ```bash
+    /usr/lib/postgresql/13.3/bin/reindexdb --all
+    ```
+
+#### Additional Information
+
+1. Estimates were performed on an 8 core 32 GB memory (t3.2xlarge EC2) machine.
+
+2. The down time for standalone scenario upgrade with 158G of unvacuumed data was about 5hr 15min.
```

Comment:
We need to make sure people understand how large this install was. We can get the node count + count for other objects like roles/envs/data bags/policyfiles/orgs/users. Otherwise everyone is going to read that and go "oh no I need 5 hours"

Reply:
I have added that along with the note